install.yaml

```
@@ -3,6 +3,7 @@
 name: ddev-drupal-core-dev
 
 project_files:
+  - config.core-dev.yaml
```

```suggestion
  - config.ddev-drupal-core-dev.yaml
```
i thought why not make it explicit and name the yaml by the actual addon name. that way it is way more clear where the config came/comes from.